### DISCUSSION TITLE
pfBlocker?

### DISCUSSION THREAD

**Discussion by tomasenskede** [lassekongo83/Frellwits-filter-lists#183] [General]
Is it possible to incorporate these filters into pfBlocker? If yes, what is the process to do so?

These configurations seem to be ineffective. Any suggestions on how to incorporate them into pfBlocker?
![image](https://github.com/lassekongo83/Frellwits-filter-lists/assets/32595351/82d9acbd-a1b2-4467-8227-bf93939fe47e)

**Comment by lassekongo83** [comment]
I don't sue pfblocker myself, but the hosts file should be compatible. Maybe you'll have to change all `127.0.0.1` to `0.0.0.0`.

The filter list is only for browser extensions.

**Comment by tomasenskede** [comment]
I possess a link to the GitHub repository for the list, ensuring its continual updates. Initial observations suggest that the filter is somewhat effective, blocking 418 requests. However, when comparing with another filter that has nearly 100,000 blocks within the same timeframe, it becomes evident that its efficiency is comparatively low.
